### PR TITLE
Deprecate section parameter for "FAQCategory" query

### LINKED ISSUE #276: Remove "section" param from FAQCategory query
- Remove "section" param from FAQCategory query - it was a silly idea 🙈 
- Do not return top-level category in `ancestors` field (as it's now) so the list doesn't contain "section" categories "Before booking", "After boooking"

First deprecate, release new version of SmartFAQ and then remove

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
Closes #276 

It was stupid.

Istead, I made root "section" categories (e.g. "After booking") invisible in subcategories.

### CODE REVIEW

File: src/FAQ/queries/FAQCategory.js [hunk 1]
```diff
@@ -30,6 +31,13 @@ const findCategory = (
   return null;
 };
 
+const sectionCategories = Object.values(sectionToCategories);
```
Comment (michalsanger) [inline src/FAQ/queries/FAQCategory.js]: Could you use `new Set(Object.values(sectionToCategories))` so the filter on line 37 can be much more faster?

File: src/FAQ/queries/FAQCategory.js [hunk 2]
```diff
@@ -42,26 +50,24 @@ export default {
...
-    { id, section }: Object,
+    { id }: Object,
     { dataLoader }: GraphqlContextType,
   ) => {
     const categoryId = Number(fromGlobalId(id).id);
-    const categories = await dataLoader.FAQCategories.load({
-      // dataloader needs to be called with value => null can't be used as default
-      section: section || 'all',
-    });
+    const categories = await dataLoader.FAQCategories.load({});
```
Comment (michalsanger) [inline src/FAQ/queries/FAQCategory.js]: Looks hacky, could only `dataLoader.FAQCategories.load()` be used?

Comment (jaroslav-kubicek) [reply src/FAQ/queries/FAQCategory.js]: nope, dataloader needs to called with value

Comment (michalsanger) [reply src/FAQ/queries/FAQCategory.js]: Yeah, but you can wrap it https://github.com/kiwicom/graphql/blob/master/src/location/dataloaders/Location.js

File: src/FAQ/queries/FAQCategory.js (hunk 1 above)
Comment (jaroslav-kubicek) [reply src/FAQ/queries/FAQCategory.js]: fixed

File: src/FAQ/queries/FAQCategory.js (hunk 2 above)
Comment (jaroslav-kubicek) [reply src/FAQ/queries/FAQCategory.js]: okey, you win _co s tebou_

Comment (jaroslav-kubicek) [reply src/FAQ/queries/FAQCategory.js]: refactored @michalsanger

**Pull request merged**